EducationPortal.Prompt/Views/Account/Profile.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using EducationPortal.Prompt.Infrastructure;
+using EducationPortal.Prompt.Views.Material.Video;
+using EducationPortal.Prompt.Views.Shared;
+
+namespace EducationPortal.Prompt.Views.Account
+{
+    public static class Profile
+    {
+        public static void Show(string msg = null)
+        {
+            Console.Clear();
+            Header.Show();
+
+            if (msg != null)
+            {
+                Console.WriteLine("========");
+                Console.WriteLine($"message: {msg}");
+                Console.WriteLine("========");
+            }
+
+            if (Provider.AuthorizedUser != null)
+            {
+                Console.WriteLine("Profile.");
+                Console.WriteLine($"Welcome, {Provider.AuthorizedUser.Login}");
+
+                Console.WriteLine("(3)Videos \n" +
```

Comment:
use StringBuilder and environment.newline

Reply:
some changed. applied